cmake/FindBLAS.cmake

```diff
@@ -569,6 +569,15 @@ if(BLA_VENDOR MATCHES "Arm" OR BLA_VENDOR STREQUAL "All")
      set(BLAS_armpl_LIB "${BLAS_armpl_LIB}_mp")
    endif()
 
+   # Arm Performance Libraries contains Fortran code and must be linked against libgfortran for a GCC build.
+   if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+     message(STATUS "gfortran")
```

Comment:
remove?

Reply:
Oh dear! Missed that, well spotted. Consider it gone!